runhouse/servers/nginx/config.py

```diff
@@ -56,8 +56,14 @@ def configure(self):
             raise RuntimeError("Failed to configure Nginx")
 
     def reload(self):
+        # If running locally (e.g. in a docker container) systemctl may not be available
+        reload_cmd = (
+            "sudo service nginx start && sudo nginx -s reload"
+            if self.address == "localhost"
```

Comment:
I wonder if we need a more robust check here.. do we need to check for 127.0.0.1 or 0.0.0.0 explicitly?